app/models/partner.rb

```diff
@@ -26,8 +27,11 @@ class Partner < ApplicationRecord
   enum status: { uninvited: 0, invited: 1, awaiting_review: 2, approved: 3, error: 4, recertification_required: 5, deactivated: 6 }
 
   belongs_to :organization
-  has_many :distributions, dependent: :destroy
+  belongs_to :partner_group, optional: true
+  has_many :item_categories, through: :partner_group
+  has_many :items, through: :item_categories
```

Comment:
not sure I like the name of this association. `partner.items` makes me think these are an items a partner has, not necessarily the ones they can request. Should we name it `requestable_items` instead?

Reply:
@albertchae good idea! I can make a quick change the association name so that it isn't as confusing.